pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
simpliest easiest is best. you can also make them mix-in's might be easier. but still keep in separate files (just don't name then ``test_*``)